### PR TITLE
Minor changes to Apple TV theme

### PR DESCRIPTION
**Pull request by Influence365**
Modified progress bar.
Small modification to shape of cards

2020-05-25-ProgressBarChange

### CODE REVIEW

File: src/themes/appletv/theme.css [hunk 1]
```diff
@@ -293,7 +293,20 @@ html {
 
 .emby-checkbox:checked + span + .checkboxOutline,
 .itemProgressBarForeground {
-    background-color: #00a4dc;
+    background: rgb(0, 210, 201);
+    background: -moz-linear-gradient(90deg, rgba(0, 210, 201, 1) 0%, rgba(13, 194, 98, 1) 28%, rgba(0, 75, 185, 1) 100%);
+    background: -webkit-linear-gradient(90deg, rgba(0, 210, 201, 1) 0%, rgba(13, 194, 98, 1) 28%, rgba(0, 75, 185, 1) 100%);
```
Comment (MrTimscampi) [inline src/themes/appletv/theme.css:298]: No need for fallbacks or vendor prefixes.

The browsers we target (Visible in `package.json` under the Browserslist key) all support CSS gradient (See https://caniuse.com/#feat=css-gradients) and we have a tool in the build process that handles vendor prefixing automatically (We've actually been meaning to remove all the prefixes we currently have).

File: src/themes/appletv/theme.css [hunk 2]
```diff
@@ -293,7 +293,20 @@ html {
...
+}
+
+.itemProgressBar {
+    background: rgba(230, 230, 230, 0.8);
+    height: 0.18rem;
+}
+
+.innerCardFooter {
+    border-radius: 0.14rem;
+    margin: 6px 8px 6px 8px;
```
Comment (MrTimscampi) [inline src/themes/appletv/theme.css:309]: Please avoid using `px` and prefer `em` or `rem`.

Comment (Influence365) [reply src/themes/appletv/theme.css:309]: Thanks, I will change

File: src/themes/appletv/theme.css (hunk 1 above)
Comment (Influence365) [reply src/themes/appletv/theme.css:298]: Thanks, I will change

**Comment by MrTimscampi** [comment]
Thank you for the changes :) This is good to go, imo.

**Pull request merged**